src/applications/facility-locator/containers/FacilitesMap.jsx

```diff
@@ -257,6 +257,9 @@ const FacilitiesMap = props => {
       zoom: MapboxInit.zoomInit,
     });
 
+    const mapContainerElement = document.getElementById('mapbox-gl-container');
+    if (mapContainerElement) mapContainerElement.setAttribute('tabindex', 0);
+
```

Comment:
I think you could accomplish the same thing by add `tabIndex={0}` to the `mapbox-gl-container` divs in `renderDesktopView` and `renderMobileView`. I think that would be slightly preferable, unless there's something I'm missing?